### DISCUSSION TITLE
Have you ever thought about creating a curl server?

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#9671] [Q&A] [answered]
Hi all. I would like to know if it would be possible to have a curl server to schedule certain things, tasks.

<br/>

*So... have you ever thought about creating a curl server?*

<br/>

**Why curl-server?**
1. The reason for this question would be to know if it would be possible to have a curl server to schedule certain things, tasks.
2. Another purpose would be to have a command history list - for example I would like to know what commands were typed in curl for all users on all local networks.
3. Another purpose would be... let's say you install curl individually on each machine. Would it be possible to have a network install with curl? how can I install curl to multiple machines on the same network?

**Comment by bagder** [comment, marked as answer]
Such ideas have been discussed, sure. I don't think they will happen any time soon though.
